cmd/policy.go

```diff
@@ -183,12 +184,23 @@ func NewPolicySys() *PolicySys {
 	}
 }
 
-func getConditionValues(request *http.Request, locationConstraint string) map[string][]string {
+func getConditionValues(request *http.Request, locationConstraint string, username string) map[string][]string {
+	currt := UTCNow()
```

Comment:
better name can be used for future readability e.g. `ctime` or `currTime` than `currt`

Reply:
Sure